### PR TITLE
apps/system/utils: Add dependency configuration fscmd

### PR DESCRIPTION
**Pull request by KimDongEon**
Changed some commands that invoke the kernel context not to be supported by not BUILD PROTECTED

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [2b74ccf4b708c51d357ded37e6370a443c7e8a93] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2b74ccf4b708c51d357ded37e6370a443c7e8a93] - Code Rule Check OK.

File: apps/system/utils/Kconfig [hunk 1]
```diff
@@ -12,7 +12,7 @@ config KERNEL_CMDS
 config FS_CMDS
 	bool "FS shell commands"
 	default y
-	depends on TASH && (NFILE_DESCRIPTORS != 0)
+	depends on !BUILD_PROTECTED && TASH && (NFILE_DESCRIPTORS != 0)
```
Comment (sunghan-chang) [inline apps/system/utils/Kconfig]: Do we need to block all of fs cmds with build_protected?

**Comment by seinfra** [comment]
Target : [3048b6c87c89123119a769946577c87da0a193e7] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [3048b6c87c89123119a769946577c87da0a193e7] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4458e76ecc000e5243acce111617e07d54fcb50a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [4458e76ecc000e5243acce111617e07d54fcb50a] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4b7df76bdd578da2345926a09c9173df8464ef04] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4b7df76bdd578da2345926a09c9173df8464ef04] - Code Rule Check (C++) OK.

File: apps/system/utils/Kconfig (hunk 1 above)
Comment (KimDongEon) [reply apps/system/utils/Kconfig]: I will modify

not depend BUILD_POTECTED : cat, cd, echo, ls, mkdir, pwd, rm, rmdir, 
depend BUILD_POTECTED : mkrd, mksmartfs, mount, umount, df

File: apps/system/utils/fscmd.c [hunk 2]
```diff
@@ -751,6 +751,7 @@ static int tash_mkdir(int argc, char **args)
 	return ret;
 }
 #endif
+#ifdef CONFIG_BUILD_PROTECTED
```
Comment (sunghan-chang) [inline apps/system/utils/fscmd.c]: Is this `ifdef`? not `ifndef`?

Comment (sunghan-chang) [reply apps/system/utils/fscmd.c]: Yeah. It means `ifndef` is correct. Please change all of `ifdef` to `ifndef`.

**Pull request closed without merging**